_ext/funkwhale.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Mopidy-Funkwhale
+type: backend
+dev:
+  gitlab: https://dev.funkwhale.audio/funkwhale/mopidy/
+dist:
+  arch-aur: mopidy-funkwhale
```

Comment:
I didn't find the package in AUR? Should this be removed too? Ref https://aur.archlinux.org/packages/?K=funkwhale

Reply:
Sure, I blindly replaced the lines *facepalm*